### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by kunalbhatia18**
for this task please look into commits of nov 18 that is the last 3
sorry i could managed to figure out a way to make a new pull request though i did make a new branch 
thank you for your support 🙂

### CODE REVIEW

File: unit-testing.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
+test-template.js
```
Comment (rajikaimal) [inline unit-testing.js]: Can you explain the purpose of this unit test?

**Comment by rajikaimal** [comment]
Still you have changes for different tasks. That being said can you explain why you have used both Jasmine and Ava.js runners.

File: unit-testing.js (hunk 1 above)
Comment (kunalbhatia18) [reply unit-testing.js]: dont look into that, it's just false commit
please look only last 3 of today's date
i dont know to delete a commit

Comment (rajikaimal) [reply unit-testing.js]: create a new branch and add only the needed changes. Look into `git reset --hard` and `git reset --soft`

**Comment by rajikaimal** [comment]
Closing this PR as this contains unwanted changes

**Pull request closed without merging**